src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/exception/BadRequestException.kt

```diff
@@ -0,0 +1,3 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.exception
+
+class BadRequestException(message: String) : RuntimeException(message)
```

Comment:
ah i see - it's not the built in bad request exception (ServerWebInputException)

this is somewhat confusing to me, since ServerWebInputException is there for the purposes of communicating http 400 bad request errors.